tests/integration-tests/tests/custom_resource/conftest.py

```diff
@@ -116,12 +116,16 @@ def cluster_1_click_fixture(cfn_stacks_factory, request, region, key_name, clust
     return stack
 
 
-def get_custom_resource_template(cluster_config_path, cluster_custom_resource_template, deletion_policy="Delete"):
+def get_custom_resource_template(
+    cluster_config_path, cluster_custom_resource_template, deletion_policy="Delete", suppress_validators=False
```

Comment:
It would be more correct and extensible to define `suppress_validator` as a list of strings (empty by default).